graphql/schema.graphql

```diff
@@ -462,6 +464,8 @@ input RepoRefInput {
   filesIgnoredFromCache: [String!]
   disabledStatsCache: Boolean
   workstationConfig: WorkstationConfigInput
+  buildBaronSettings: BuildBaronSettingsInput
```

Comment:
Do these additional inputs require changes to [SaveProjectSettingsForSection](https://github.com/evergreen-ci/evergreen/blob/main/rest/data/project_settings.go#L78) for them to processed as part of the saveRepoSettingsForSection mutation?

Reply:
Yes, that has been done in this PR: https://github.com/evergreen-ci/evergreen/pull/5154